### ISSUE TITLE
AppServiceDemo on rn-windows 0.66.3 does not build

### ISSUE THREAD

**Issue by tero-paananen** [microsoft/react-native-windows-samples#586]
### Sample

AppServiceDemo

### Problem Description

AppServiceDemo on `rn-windows 0.66.3` does not build. I get error `Payload contains two or more files with the same destination path 'Microsoft.UI.Xaml.Markup.winmd'`  

I have same problem with my app that is React Native Windows 0.66.2 UWP C# and Win32 console solution.

See AppServiceDemo example app ported into 0.66.3 below.

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	APPX1112	Payload contains two or more files with the same destination path 'Microsoft.UI.Xaml.Markup.winmd', but they have different content. Source files: 
C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\XamlCompiler\Microsoft.UI.Xaml.Markup.winmd
C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\XamlCompiler\Microsoft.UI.Xaml.Markup.winmd	appservicedemo.Package	C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VisualStudio\v16.0\AppxPackage\Microsoft.AppXPackage.Targets	1946	

```

### Steps To Reproduce

Use AppServiceDemo and port it into rn-windows 0.66.3

### CLI version

6.0.0

### Environment

```markdown
System:
    OS: Windows 10 10.0.19043
    CPU: (12) x64 Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
    Memory: 21.61 GB / 31.90 GB
  Binaries:
    Node: 12.22.5 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.5 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.14.14 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
    Windows SDK:
      AllowDevelopmentWithoutDevLicense: Enabled
      AllowAllTrustedApps: Enabled
      Versions: 10.0.14393.0, 10.0.18362.0, 10.0.19041.0
  IDEs:
    Android Studio: Not Found
    Visual Studio: 16.11.31624.102 (Visual Studio Community 2019)
  Languages:
    Java: Not Found
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2
    react-native: 0.66.1 => 0.66.1
    react-native-windows: 0.66.3 => 0.66.3
  npmGlobalPackages:
    *react-native*: Not Found
```

### Target Device(s)

Desktop

### Visual Studio Version

Visual Studio 2019 16.11.2

### Build Configuration

Debug

**Comment by tero-paananen** [comment]
[AppServiceDemo_0.66.zip](https://github.com/microsoft/react-native-windows-samples/files/7474492/AppServiceDemo_0.66.zip)

There is AppServiceDemo that uses rn-windows 0.66.3

**Comment by tero-paananen** [comment]
I tried use same `<TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>` versions in `appservicedemo.Package.wapproj`  and `appservicedemo.csproj`  but did not help

**Comment by tero-paananen** [comment]
> I tried use same `<TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>` versions in `appservicedemo.Package.wapproj` and `appservicedemo.csproj` but did not help

After setting also same version into `ReactNativeAppServiceModule.csproj` project builds.

**Comment by tero-paananen** [comment]
So if solution has same `<TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>`  for all sub-projects it will build.

**Comment by asklar** [comment]
0.66 changed the target platform version to 19041: microsoft/react-native-windows#7779
So this may be a case of "upgrading just the entry in package.json isn't enough".
See release notes for 0.66: https://github.com/microsoft/react-native-windows/releases/tag/react-native-windows_v0.66.0
> Windows 10 SDK version 10.0.19041.0: We now use the latest Windows SDK, included with Visual Studio. Note that you may still need to install previous SDKs to use community modules that have not yet updated.

Closing since this seems to be unblocked : )

**Comment by tero-paananen** [comment]
"Note that you may still need to install previous SDKs to use community modules that have not yet updated." seems not work if we have to update versions in all modules in our project

**Comment by NickGerleman** [comment]
I would also expect older community modules to work, like they do in gallery. It sounds like there may be a specific issue with this example...

@jonthysell is there work to upgrade the samples?

**Comment by tero-paananen** [comment]
This example and my app is package project that contains app and win32 exe. I think that it causes that issue

**Comment by tero-paananen** [comment]
Issue exists still, i update to latest Visual Studio 16.11.6.

**Comment by tero-paananen** [comment]
To get package solution build with latest VS 16.11.6 i had to do some more "hacking"


https://developercommunity.visualstudio.com/t/visual-studio-2019-version-1610uwpxamarin-runtime/1437211

https://github.com/microsoft/microsoft-ui-xaml/issues/6195

I hope that VS 2022 might work better for this kind of projects.

**Comment by tero-paananen** [comment]
Sorry to keep conversation in closed issue - i am not sure if conversation after close not allowed

**Comment by chrisglein** [comment]
> Sorry to keep conversation in closed issue - i am not sure if conversation after close not allowed

Not disallowed, just not as sure to get a response. We do have a bot that will lock stale closed issues, but we aren't there yet.

It looks like this was closed because we thought you were unblocked, but that doesn't seem to be the the case. Is that correct? If so, we can either reactivate or you can open a new issue (if indeed something was resolved but you're not hitting a new blocker further down the journey).

**Comment by tero-paananen** [comment]
> > Sorry to keep conversation in closed issue - i am not sure if conversation after close not allowed
> 
> Not disallowed, just not as sure to get a response. We do have a bot that will lock stale closed issues, but we aren't there yet.
> 
> It looks like this was closed because we thought you were unblocked, but that doesn't seem to be the the case. Is that correct? If so, we can either reactivate or you can open a new issue (if indeed something was resolved but you're not hitting a new blocker further down the journey).

Reactivate this issue please. I have still problem to get this kind of solutions on 0.66 to build with for example VS 16.11.6.
